### PR TITLE
Mock data

### LINKED ISSUE #10: Get data into challenges plugin
Using Marie's script, populate db associated with this plugin with mock data.

### PR DESCRIPTION
**Pull request by JMurphyWeb**
closes #10 

I had to change the file a bit:
- now initialises tags, categories and tags_categories tables
- now populates above tables
- now export the function in case we want to use in tests
- now reads config from the single config file

### CODE REVIEW

**Comment by nelsonic** [comment]
@JMurphyWeb please _ensure_ the build is passing _before_ assigning for review. thanks. 👍

File: fixtures/initialise_tables.sql [hunk 1]
```diff
@@ -0,0 +1,74 @@
+-- Delete tables
+DROP TABLE IF EXISTS tags_challenges CASCADE;
+DROP TABLE IF EXISTS challenges CASCADE;
+DROP TABLE IF EXISTS tags_organisations CASCADE;
+DROP TABLE IF EXISTS organisations CASCADE;
+DROP TABLE IF EXISTS people CASCADE;
+
+-- TAGS --
```
Comment (SimonLab) [inline fixtures/initialise_tables.sql]: Deleting the tags and recreating them inside the challenges plugin will create some conflict if when we will use the challenges plugin with the tags plugin. If we need some tags to test the challenges plugin can we just register the tags plugin which will create the right tables?

Comment (JMurphyWeb) [inline fixtures/initialise_tables.sql]: It not run on registration. It is an external initialiser, similar to what we have on ce100, with the addition of tags

Comment (SimonLab) [inline fixtures/initialise_tables.sql]: :+1:

**Pull request merged**